src/js/player.js

```diff
@@ -454,12 +518,119 @@ class Player extends Component {
    * Fired when the player switches in or out of fullscreen mode
    * @event fullscreenchange
    */
-  onFullscreenChange() {
+  onTechFullscreenChange() {
     if (this.isFullscreen()) {
       this.addClass('vjs-fullscreen');
     } else {
       this.removeClass('vjs-fullscreen');
     }
+
+    this.trigger('fullscreenchange');
+  }
+
+  /**
+   * Fires when an error occurred during the loading of an audio/video
+   * @event error
+   */
+  onTechError() {
+    this.error(this.tech.error().code);
+    this.trigger('error');
```

Comment:
Calling `player.error(err)` already triggers the player error event, so this would be triggering a second. We should probably be more consistent with how we handle states like this, so we're adding css classes and triggering events the same way for all of them. We don't need to figure that out in this PR specifically though.